integration/docker-compose/docker-compose.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Docker compose version
+VERSION="1.20.0"
+# URL
+URL="http://localhost:5000"
+# Image name
+IMAGE_NAME="dockercompose_web"
+# Installation path
+INSTALLATION_PATH="/usr/local/bin/docker-compose"
+
+setup () {
+	which docker-compose && exit
+	# Check that Docker compose is installed in the system
+	if [ $? -ne 0 ]; then
+		echo >&2 "ERROR: Docker compose should be installed in the system."
+		# Install docker-compose
+		echo >&2 "Installing Docker compose."
+		curl -L https://github.com/docker/compose/releases/download/${VERSION}/docker-compose-`uname -s`-`uname -m` -o ${INSTALLATION_PATH}
+		sudo -E chmod +x ${INSTALLATION_PATH}
+	fi
+
+	# Run the services in the background
+	docker-compose up -d
+}
+
+@test "run REDIS application" {
+	skip "This is not working (https://github.com/kata-containers/tests/issues/108)"
```

Comment:
I think the issue here is: https://github.com/clearcontainers/runtime/issues/1042